css/tokens/index.js

```diff
@@ -1,30 +1,38 @@
-const sassExtract = require('sass-extract');
 const fs = require('fs-extra');
 const path = require('path');
-const { convertSassVariable } = require('./conversion');
 const { quicktype, InputData, jsonInputForTargetLanguage } = require('quicktype-core');
+const { exporter } = require('sass-export');
 
 createTokens();
 
 async function createTokens() {
-	const indexFile = require.resolve('../src/tokens/index.scss');
+	const options = {
+		inputFiles: [
```

Comment:
In theory, I should be able to include only the index file and add the includePaths option with the tokens root path to resolve the imports in index.scss. However, I tried every path combination that I could think of and couldn't get it to work.

Reply:
I played around an encountered some of the same issues that you did. I don't see a major issue with this set up, but I'd like to avoid a few things:

1. every time a token file is added, we have to think about this build array
2. every time we add a token file, we may forget the required comments.

So what about this, we instead use [`fast-glob`](https://www.npmjs.com/package/fast-glob
) to get the inputFiles in a more programatic way?

```js
const glob = require('fast-glob');

createTokens();

async function createTokens() {
	const include = './src/tokens/*.scss';
	const ignore = ['./src/tokens/index.scss'];

	console.log({ include }); // testing only
	console.log({ ignore }); // testing only
	const filePaths = await glob(include, {
		ignore
	});

	console.log({ filePaths }); // testing only
	[...]
```

For item 2, we can deal with that as a follow up perhaps, but I think the logic would be something similar to this: 
If a file is not in the ignore array, but does not contain the required sass-export comments, we should show a warning in the console.

Reply:
Yes I didn't like the idea of encountering the two issues either. I'll give fast-glob a try.

Reply:
The only problematic token file is `palette` since` colors` comes first and uses palette tokens. I added palette path first to the files list, then the paths from fast-glob.